src/pages/ReimbursementAccount/BusinessInfo/substeps/ConfirmationBusiness.tsx

```diff
@@ -48,17 +32,28 @@ const validate = (values) => {
     return errors;
 };
 
-function ConfirmationBusiness({reimbursementAccount, reimbursementAccountDraft, onNext, onMove}) {
+type ConfirmationBusinessOnyxProps = {
+    reimbursementAccount: OnyxEntry<ReimbursementAccount>;
+    reimbursementAccountDraft: OnyxEntry<ReimbursementAccountDraft>;
+};
+
+type ConfirmationBusinessProps = {
+    reimbursementAccount: ReimbursementAccount;
+    reimbursementAccountDraft: ReimbursementAccountDraft;
+} & SubStepProps;
```

Comment:
Also, please add comments describing params if possible.
```suggestion
type ConfirmationBusinessOnyxProps = {
    reimbursementAccount: OnyxEntry<ReimbursementAccount>;
    reimbursementAccountDraft: OnyxEntry<ReimbursementAccountDraft>;
};

type ConfirmationBusinessProps = ConfirmationBusinessOnyxProps & SubStepProps;
```